vr-je-palette-picker/palette-picker-back-end
# Migrate initial

## What does this PR do?
This PR has the initial and latest migrations creating two tables, projects and palettes, with the appropriate columns. This PR also includes data for seeding.

### Where should the reviewer start?
`knex.js`, `migrations` directory, and `data` directory. 

#### How should this be manually tested?
Start up server. In the terminal, or in an application like Postico, within the `palette-picker` database, check for column names.

#### Any background context you want to provide?


#### What are the relevant tickets?
User Story: Projects Table, and User Story: Palettes Table

Closes: #32 #31 

#### Screenshots (if appropriate)


#### Additional Notes
